libs/rollup.config.ts

```diff
@@ -239,6 +239,27 @@ export const outputConfigs = {
     external
   },
 
+  /**
+   * Libs Web Package
+   * Used by the Audius Web Client as a more performant/smaller bundle
+   */
+  webConfig: {
```

Comment:
Soooo many rollup bundles, but at least we can remove the legacy one once this is proven to work right?

Reply:
i know right?? yes hopefully should just be a replacement long-term :)